app/views/modules/_header.html.erb

```diff
@@ -26,6 +26,7 @@ Unless required by applicable law or agreed to in writing, software distributed
     <div class="col-sm-4">
       <form accept-charset="UTF-8" action="/catalog" id="search-form-header" method="get" role="search" class="form-inline hidden-xs mast-sm-search">
         <div style="display:none"><input name="utf8" type="hidden" value="✓"></div>
+        <input id="search_field" name="search_field" type="hidden" value="all_fields">
```

Comment:
The thing runs fine if the tag is omitted, but this test fails:

https://github.com/ualbertalib/avalon/blob/master/spec/features/capybara_navigation_spec.rb#L76

I had a choice between changing the code or changing the test, so I chose to change the code to match upstream.

The code for the search box was originally here, but was moved in the local version to put it in the header:

https://github.com/ualbertalib/avalon/blob/master/app/views/modules/_avalon_search_form.html.erb#L21